components/body/index.styl

```diff
@@ -65,22 +87,36 @@
 
 .section-two-text
   margin-bottom 40px
+  p
+    font-size 16px
 
-.section-two-text  p
-  font-size 16px
-
-.section-three-content p, .section-four-content p
-  margin-bottom 60px
-  line-height 1.5
-
-// .section-three-content
-  // display none
-
-.launch-app-store
-  width 205px
+.avant-garde-button-black
```

Comment:
Styles like this and `.carousel-track__image` are legit, but they belong outside of the body component b/c they're used outside of the scope of just the body (or if they're aren't then they should be namespaced by `body`). You could move these to stylesheets inside the "global" layout component e.g. components/layout/buttons.styl components/layout/carousel.styl (what we do in Force/MG) or extract them as their own components e.g. components/buttons/index.styl, components/carousel/index.styl

Not a big deal, but just hoping erring on overly scrutinizing is helpful in understanding how we organize components.